app/javascript/components/visualization/controls/ClusterSelector.js

```diff
@@ -50,7 +50,7 @@ export default function ClusterSelector({
 
   return (
     <div className="form-group">
-      <label className="labeled-select">Clustering
+      <label className="labeled-select">Clustering2
```

Comment:
Is this supposed to say `Clustering2`, or was this just a debug thing?

Reply:
leftover debug, good catch